core/src/main/java/org/apache/accumulo/core/data/Value.java

```diff
@@ -48,6 +50,30 @@ public Value() {
   }
 
   /**
+   * Creates a value using the UTF-8 encoding of the CharSequence
+   *
+   * @param cs
+   *          may not be null
+   *
+   * @since 1.8.0
+   */
+  public Value(CharSequence cs) {
+    this(cs.toString().getBytes(StandardCharsets.UTF_8));
+  }
+
+  /**
+   * Creates a Value using the bytes of the Text. Makes a copy, does not use the byte array from the Text.
+   *
+   * @param text
+   *          may not be null
+   *
+   * @since 1.8.0
+   */
+  public Value(Text text) {
+    this(text.getBytes(), 0, text.getLength());
```

Comment:
IMO, you doc'ed it and there is a way for users to do a non-copy constructor with Text. That's both of the things I would be worried about :)